runner/providers/provider-github.ts

```diff
@@ -42,14 +45,23 @@ export async function getRepository(project: Project) {
   }
 }
 
-export async function getRepositories(config: Config) {
+export async function getRepositories(
+  config: Pick<
+    Config,
+    'organization' | 'amountOfRepos' | 'repositories' | 'excludeRepos'
+  >
+) {
   const { organization } = await client.request(QUERY_GITHUB_REPOSITORIES, {
     organization: config.organization,
     amountOfRepos: config.amountOfRepos ?? DEFAULT_AMOUNT_OF_REPOS,
   })
-  const orgRepositories: string[] = organization.repositories.nodes.map(({ name }: { name: string }) => `${config.organization}/${name}`)
+  const orgRepositories: string[] = organization.repositories.nodes.map(
+    ({ name }: { name: string }) => `${config.organization}/${name}`
+  )
   const repos = config.repositories ?? orgRepositories ?? []
   const exclude = config.excludeRepos ?? []
 
-  return repos.filter(repo => exclude.every(exclusion => !repo.includes(exclusion)))
+  return repos.filter((repo) =>
```

Comment:
I suggest adding `isArchived` to the organization query and filtering repos before doing any individual requests.

Reply:
It's already done, but in `scanProjects` function in https://github.com/monito/technical-stats/blob/main/runner/index.ts